<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Tela_Principal</class>
 <widget class="QMainWindow" name="Tela_Principal">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>926</width>
    <height>547</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>80</y>
      <width>241</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Lucida Sans Unicode</family>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="ver_acervo">
    <property name="geometry">
     <rect>
      <x>500</x>
      <y>260</y>
      <width>291</width>
      <height>81</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Lucida Sans Unicode</family>
      <pointsize>9</pointsize>
     </font>
    </property>
    <property name="text">
     <string>VER ACERVO</string>
    </property>
    <property name="icon">
     <iconset>
      <normaloff>images/lupa.png</normaloff>images/lupa.png</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>25</width>
      <height>25</height>
     </size>
    </property>
   </widget>
   <widget class="QTextBrowser" name="textBrowser">
    <property name="geometry">
     <rect>
      <x>-10</x>
      <y>-20</y>
      <width>941</width>
      <height>571</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="cadastrar_novo_livro">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>260</y>
      <width>291</width>
      <height>81</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Lucida Sans Unicode</family>
      <pointsize>9</pointsize>
     </font>
    </property>
    <property name="text">
     <string>CADASTAR NOVO LIVRO</string>
    </property>
    <property name="icon">
     <iconset>
      <normaloff>images/new-file_40454.png</normaloff>images/new-file_40454.png</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>25</width>
      <height>25</height>
     </size>
    </property>
   </widget>
   <widget class="QPushButton" name="sair">
    <property name="geometry">
     <rect>
      <x>700</x>
      <y>90</y>
      <width>93</width>
      <height>28</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Lucida Sans Unicode</family>
      <pointsize>9</pointsize>
     </font>
    </property>
    <property name="text">
     <string>SAIR</string>
    </property>
    <property name="icon">
     <iconset>
      <normaloff>images/exit.png</normaloff>images/exit.png</iconset>
    </property>
   </widget>
   <zorder>textBrowser</zorder>
   <zorder>label</zorder>
   <zorder>ver_acervo</zorder>
   <zorder>cadastrar_novo_livro</zorder>
   <zorder>sair</zorder>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>926</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
